server/server.ts

```diff
@@ -1,8 +1,25 @@
 /* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
 import 'dotenv/config';
 import pg from 'pg';
-import express from 'express';
-import { ClientError, errorMiddleware } from './lib/index.js';
+import express, { application } from 'express';
+import { authMiddleware, ClientError, errorMiddleware } from './lib/index.js';
+import { type Entry } from '../client/src/data.js';
```

Comment:
We usually don't import client code in server, and vice versa. That's because they are usually in separate repos and don't have access to each others' code. In this case, though, it works.